benchmark/bm_entangling_layers.py

```diff
@@ -26,9 +26,8 @@
 
 
 def circuit(weights, *, features=None):
-    """Immutable quantum circuit."""
+    """Mutable quantum circuit."""
```

Comment:
@johannesjmeyer, what is the reasoning for this change here?

Reply:
The test failed (I guess due to the new quantum function rules?) so I fixed it.

Reply:
I'm happy to have it go back to immutable, but I don't think this is in scope? @smite?

Reply:
@trbromley I think also reported this at one point(?)

Reply:
Yes I also had a problem with the immutable version, I think @smite said there's a planned fix here: https://github.com/XanaduAI/pennylane/pull/510